packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -110,8 +110,20 @@ class ScrollBehavior {
   /// {@macro flutter.gestures.monodrag.DragGestureRecognizer.multitouchDragStrategy}
   ///
   /// By default, [MultitouchDragStrategy.latestPointer] is configured to
-  /// create drag gestures for all platforms.
-  MultitouchDragStrategy get multitouchDragStrategy => MultitouchDragStrategy.latestPointer;
+  /// create drag gestures for non-Apple platforms, and
+  /// [MultitouchDragStrategy.maxAllPointers] for Apple platforms.
+  MultitouchDragStrategy get multitouchDragStrategy {
```

Comment:
I wonder if this should not be converted to a method so that is can have a BuildContext?

The defaultTargetPlatform is used by ScrollBehavior, but subclasses like MaterialScrollBehavior and CupertinoScrollBehavior use the getPlatform method using context to account for if the Theme is specifying a platform.

Reply:
Good point.
I'm worried that this will become a breaking change unless we can guarantee that this will be released at the same time as the previous PR. What do you think?

Reply:
They won't be released at the same time. The other PR has already been cut into multiple beta releases I think. 
It feels like a worthwhile breaking change though with low risk. Not a lot of folks will have had the opportunity to use the new getter, so following up with a small change to make it a method should be ok. I na help with a migration guide.

Reply:
I think dart fix will be able to help too.